content/tracing/product_specs/distributed_tracing.md

```diff
@@ -0,0 +1,162 @@
+    from ddtrace.contrib.flask import TraceMiddleware
+
+    ## Configuring Datadog tracer
+    app = Flask('API')
+    traced_app = TraceMiddleware(app, tracer, service='thinker-api')
+
+    @app.route('/think/')
+    def think_handler():
+        thoughts = requests.get('http://thinker:8000/', headers={
+            'x-datadog-trace-id': str(tracer.current_span().trace_id),
+            'x-datadog-parent-id': str(tracer.current_span().span_id),
+        }, params={
+            'subject': flask_request.args.getlist('subject', str),
+        }).content
+        return Response(thoughts, mimetype='application/json')
+
+    ```
+
+* **Thinker Microservice**: Takes a request from **thinker-api** with one or multiple *subjects* and answers with a *thought* if the *subject* is "technology":
+    ```python
+    import asyncio
+
+    from aiohttp import web
+    from ddtrace import tracer
+    from ddtrace.contrib.aiohttp import trace_app
+
+    app = web.Application()
+    app.router.add_get('/', handle)
+
+    trace_app(app, tracer, service='thinker-microservice')
+    app['datadog_trace']['distributed_tracing_enabled'] = True
+
+    @tracer.wrap(name='think')
+    async def think(subject):
+        tracer.current_span().set_tag('subject', subject)
+
+        await asyncio.sleep(0.5)
+        return thoughts[subject]
+
+    thoughts = {
+        'technology': Thought(
+            quote='For a successful technology, reality must take precedence over public relations, for Nature cannot be fooled.',
+            author='Richard Feynman',
+        ),
+    }
+
+    async def handle(request):
+        response = {}
+        for subject in request.query.getall('subject', []):
+            await asyncio.sleep(0.2)
+            try:
+                thought = await think(subject)
+                response[subject] = {
+                    'error': False,
+                    'quote': thought.quote,
+                    'author': thought.author,
+                }
+            except KeyError:
+                response[subject] = {
+                    'error': True,
+                    'reason': 'Subject unknown'
+                }
+
+        return web.json_response(response)
+    ```
+
+The code above is already instrumented. Please [refer to the dedicated setup documentation](/tracing/setup) to learn how to instrument your application and configure the Datadog Agent.
+
+### Datadog APM
+
+Once this code is executed, [traces](tracing/services/trace/) and statistics are collected by the Agent and send to a Datadog platform in the [services list page](/tracing/services) where there are now two services: **thinker-api** and **thinker-microservice** reporting.
+
+{{< img src="tracing/miscellaneous/getting_started_distributed/services_GS.png" alt="Services list getting started" responsive="true" popup="true">}}
+
+Clicking on **thinker-api** service, directs you to its dedicated [service page](/tracing/services/service), where there are:
```

Comment:
"Clicking on **thinker-api** directs you to it's automatically generated service dashboard.  Here we can see more detailed performance data, as well as a list of all of the resources associated with this particular service."

Let's just show an image of the dash below instead of linking off the page.

Reply:
The dash for this example is pretty empty since it's a "dummy" example. Linking to the service dash on the service page avoid duplication of content.